week4/src/components/StoreDataCard.jsx

```diff
@@ -0,0 +1,80 @@
+import styled from "styled-components";
+import Skeleton from "./Skeleton";
+
+function StoreDataCard({ isLoading, data, isChecked }) {
+  return (
+    <>
+      {isLoading
+        ? new Array(10).fill(1).map((_, idx) => {
+            return <Skeleton key={idx} />;
+          })
+        : data &&
+          data.map((beerstore, idx) => {
+            return (
+              <Card key={idx}>
+                <CardTitle href={beerstore.place_url || null}>
+                  {beerstore.place_name}
+                </CardTitle>
+                <InfoBox>
+                  <p className="info__tel">{beerstore.phone || "번호 없음"}</p>
+                  {!isChecked ? (
+                    <p className="info__address">{beerstore.distance}m</p>
+                  ) : (
+                    <p className="info__address">{beerstore.address_name}</p>
+                  )}
+                </InfoBox>
+              </Card>
+            );
+          })}
```

Comment:
`beerstore.`이 계속 쓰이고 있으니까 구조 분해 할당 이렇게 하면 좋을 것 같아 !!

```suggestion
          data.map(({place_url, place_name, phone, distance, address_name}, idx) => {
            return (
              <Card key={idx}>
                <CardTitle href={place_url || null}>
                  {place_name}
                </CardTitle>
                <InfoBox>
                  <p className="info__tel">{phone || "번호 없음"}</p>
                  {!isChecked ? (
                    <p className="info__address">{distance}m</p>
                  ) : (
                    <p className="info__address">{address_name}</p>
                  )}
                </InfoBox>
              </Card>
            );
          })}
```

Reply:
오 훨씬 깔끔하다!! 고마워 앞으로도 이런 식으로 써야겠다